testcases/testcase1/models/Resources/emissions.csv

```diff
@@ -1,3 +1,10 @@
+# Optional header information
+# time: time of emissions factor [s]
+# EmissionsElectricPower: Emissions factor of electric poweer [kgCO2-e/kWh]
```

Comment:
I'm not familiar with the `-e` notation. Does e stand for emission?

Reply:
CO2 equivalent.

Reply:
okay!